BDArmory/FX/FuelLeakFX.cs

```diff
@@ -55,8 +55,8 @@ void OnEnable()
             BDArmorySetup.numberOfParticleEmitters++;
             startTime = Time.time;
             pEmitters = gameObject.GetComponentsInChildren<KSPParticleEmitter>();
-            bool vacuum = FlightGlobals.getAtmDensity(FlightGlobals.getStaticPressure(transform.position),
-            FlightGlobals.getExternalTemperature(), FlightGlobals.currentMainBody) < 0.05f;
+            bool vacuum = false;
+            if (parentPart.vessel.situation == Vessel.Situations.ORBITING || parentPart.vessel.situation == Vessel.Situations.SUB_ORBITAL) vacuum = true;
```

Comment:
`parentPart.vessel.InOrbit()` does this check, plus checks for escaping trajectories. However, you can be on a sub-orbital trajectory while still in atmosphere.
Wouldn't the usual `parentPart.vessel.atmDensity < 0.05f` check be more appropriate? (Or `parentPart.vessel.InVacuum()`, which apparently uses `0.001f`?)

Reply:
Generally, yes. I initially set it up that way so things like tank battles on the Mun would still have gravity properly affect the FX, but thinking about it, that would still require a velocity element as well, so swapping over to InVacuum() for now since that does make more sense for the time being.